db/schema.rb

```diff
@@ -22,19 +21,19 @@
     t.bigint "resource_id"
     t.string "author_type"
     t.bigint "author_id"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
-    t.index ["author_type", "author_id"], name: "index_active_admin_comments_on_author_type_and_author_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_type", "author_id"], name: "index_active_admin_comments_on_author"
```

Comment:
care the `db:migrate` command generate this diff 
![Screenshot from 2025-05-31 17-25-45](https://github.com/user-attachments/assets/70bea955-9d14-4919-9a4a-4795f3054110)

Reply:
Yes, I started all migrations from a fresh DB just to be sure.
Although doing that revealed that a few more changes were needed (explained in the commit messages)